Documentation/tutorial/advanced_support_hyper_link.md

```diff
@@ -0,0 +1,111 @@
+﻿🔧 Advanced: Support Hyper Link
+===============================
+
+In this topic, we will support hyper link in rtf files.
+
+Agenda
+------
+* [Rules for hyper link](#rules-for-hyper-link)
+* [Prepare](#prepare)
+* [Update rtf document processor](#update-rtf-document-processor)
+* [Test and verify](#test-and-verify)
+
+Rules for hyper link
+--------------------
+1.  For relative path, always from working folder, i.e. start with `~/`.
+2.  Do NOT contain `..` or `//` in parts.
+3.  Do NOT use `\`.
+4.  Report link while saving.
+5.  ONLY report relative path and from working folder.
+6.  Do NOT encode while reporting.
+7.  Do NOT report anchor.
+
+Prepare
+-------
+1.  Open the rtf plug-in library project in `Visual Studio`.
+
+2.  Add nuget packages:  
+    for plug-in: `Microsoft.DocAsCode.Utility`
+
+3.  Add framework assembly reference:
+    `System.Core`, `System.Web`, `System.Xml.Linq`
+
+Update rtf document processor
+-----------------------------
+1.  Following rules for hyper link, add `FixLink` help method:
+    ```c#
```

Comment:
how about put all code snippets to a workable plugin project and use our code snippet extension to reference them?